frontend/packages/otr/src/components/clerkInterpreter/overview/ClerkInterpreterDetails.tsx

```diff
@@ -166,10 +167,10 @@ export const ClerkInterpreterDetails = () => {
   useNavigationProtection(hasLocalChanges);
 
   const hasRequiredDetails =
-    !!interpreterDetails?.firstName &&
-    !!interpreterDetails.lastName &&
-    !!interpreterDetails.nickName &&
-    !!interpreterDetails.email;
+    StringUtils.isNonBlankString(interpreterDetails?.lastName) &&
+    StringUtils.isNonBlankString(interpreterDetails?.firstName) &&
+    StringUtils.isNonBlankString(interpreterDetails?.nickName) &&
+    StringUtils.isNonBlankString(interpreterDetails?.email);
```

Comment:
Sinänsä tuo interpreterDetails on jo näissä kolmessa jälkimmäisessä ehdossa pakosti olemassa => `interpreterDetails?.` voi olla ihan vain `interpreterDetails.`. Saa kyllä jäädä näinkin.

Reply:
Nyt kun tuo `interpreterDetails?.lastName` annetaan metodille parameterina herjaa mulla ainakin IntellJ, että `interpreterDetails` voi olla undefined

Reply:
Ok. Jääköön näin.